<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureDialog</class>
 <widget class="QDialog" name="TextureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>532</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Textures Browser</string>
  </property>
  <property name="statusTip">
   <string>Ready</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textureFilterEdit">
           <property name="toolTip">
            <string>Type here to search</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearFilterButton">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Clear search results</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="TextureList" name="listViewTextures">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolBar" name="textureListToolbar">
         <addaction name="actionViewImagesList"/>
         <addaction name="actionViewTextList"/>
         <addaction name="separator"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitterH">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>80</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frameTextures">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QToolBar" name="textureToolbar">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="floatable">
           <bool>true</bool>
          </property>
          <addaction name="actionColorR"/>
          <addaction name="actionColorG"/>
          <addaction name="actionColorB"/>
          <addaction name="actionColorA"/>
          <addaction name="separator"/>
          <addaction name="actionZoomIn"/>
          <addaction name="actionZoomOut"/>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="TextureScrollArea" name="textureAreaOriginal">
             <property name="cursor" stdset="0">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="widgetResizable" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolBar" name="textureViewToolbar">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <addaction name="actionViewPVR"/>
             <addaction name="actionViewDXT"/>
            </widget>
           </item>
           <item>
            <widget class="TextureScrollArea" name="textureAreaPVR">
             <property name="cursor" stdset="0">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="widgetResizable" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="labelOriginal">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelConverted">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="TextureProperties" name="textureProperties" native="true"/>
    </widget>
   </item>
  </layout>
  <action name="actionColorR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>colorR</string>
   </property>
  </action>
  <action name="actionColorG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>colorG</string>
   </property>
  </action>
  <action name="actionColorB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>colorB</string>
   </property>
  </action>
  <action name="actionColorA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>colorA</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>zoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>zoomOut</string>
   </property>
  </action>
  <action name="actionViewImagesList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>viewImagesList</string>
   </property>
  </action>
  <action name="actionViewTextList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>viewTextList</string>
   </property>
  </action>
  <action name="actionViewPVR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PVR</string>
   </property>
   <property name="toolTip">
    <string>PVR</string>
   </property>
  </action>
  <action name="actionViewDXT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DXT</string>
   </property>
   <property name="toolTip">
    <string>DXT</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextureScrollArea</class>
   <extends>QGraphicsView</extends>
   <header>texturescrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextureList</class>
   <extends>QListView</extends>
   <header>texturelist.h</header>
  </customwidget>
  <customwidget>
   <class>TextureProperties</class>
   <extends>QWidget</extends>
   <header>textureproperties.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>clearFilterButton</sender>
   <signal>clicked()</signal>
   <receiver>textureFilterEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
